asn1/InterledgerPaymentRequest.asn

```diff
@@ -0,0 +1,21 @@
+InterledgerProtocol
+DEFINITIONS
+AUTOMATIC TAGS ::=
+BEGIN
+
+IMPORTS
+    UInt8
+    UInt256
+    FROM GenericTypes
+;
+
+InterledgerPaymentRequest ::= SEQUENCE {
+    -- IPR Version
+    version UInt8,
+    -- ILP Payment Packet
+    packet OCTET STRING,
+    -- Cryptographic condition
+    condition UInt256
+}
```

Comment:
We _might_ want to add an expiry to this format to let the sender know how long they have to execute it. The packet in the IPR may have an expiry in the data field but the current spec says the sender must treat the data part of the packet as opaque.